cmd/node/config/economics.toml

```diff
@@ -14,20 +27,20 @@
 [FeeSettings]
     MaxGasLimitPerBlock     = "1500000000"
     MaxGasLimitPerMetaBlock = "15000000000"
-    MinGasPrice = "200000000000" #will yield min tx fee of 0.01ERD
+    MinGasPrice = "500000000" #will yield min tx fee of 0.01ERD
```

Comment:
the comment does not reflect the value. 5 -> 2?

Reply:
done